opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -51,12 +51,47 @@ static char *ofi_progress_mode;
 static bool disable_sep;
 static int mca_btl_ofi_init_device(struct fi_info *info);
 
+static int
```

Comment:
This code is exactly the same in the MTL and BTL.  This should be moved to common and only exist once.

Reply:
Makes sense, will do that.